gammapy/maps/region/tests/test_ndmap.py

```diff
@@ -375,3 +376,35 @@ def test_region_nd_hdulist():
     assert hdulist[1].name == "SKYMAP"
     assert hdulist[2].name == "SKYMAP_BANDS"
     assert hdulist[3].name == "SKYMAP_REGION"
+
+def test_region_map_sampling(region_map):
+    energy_axis = MapAxis.from_energy_bounds("1 TeV", "10 TeV", nbin=6, name="energy")
+    edges = np.linspace(0., 30., 3)*u.min
+    time_axis = MapAxis.from_edges(edges, name="time")
+
+    npred_map = Map.create(
+        region="icrs;circle(83.63, 21.51, 1)",
+        map_type="region",
+        axes=[time_axis, energy_axis],
+    )
+    npred_map.data[...] = 5
+
+    coords = npred_map.sample_coord(n_events=2, random_state=0)
+    skycoord = coords.skycoord
+
+    events = Table()
```

Comment:
Why create the table here? I guess it was just copy and paste? Ok, to keep but could be removed as well for this test...